composer.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "open-runtimes/proxy",
+    "description": "Serverless runtimes proxy to split traffic betwen executors ⚡️",
+    "type": "project",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "OpenRuntimes\\Executor\\": "src/OpenRuntimes/Executor"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Tests\\": "tests"
+        }
+    },
+    "scripts": {
+        "lint": "./vendor/bin/pint --preset psr12 --test",
+        "format": "./vendor/bin/pint --preset psr12",
+        "check": "./vendor/bin/phpstan analyse --level 8 app tests",
+        "test": "./vendor/bin/phpunit --configuration phpunit.xml --debug"
+    },
+    "require": {
+        "php": ">=8.0.0",
+        "ext-curl": "*",
+        "ext-imagick": "*",
```

Comment:
```suggestion
```

Reply:
While at it, what about `json` and `mbstring`? I copy&pasted this from other composer, not entirely sure what functions it adds and if we use them.

Reply:
`json` we probably need. `mbstring` not sure. both are PHP extensions that give more built-in core functionality. I think some of the autocompletion is relying on these detentions, not 100% sure.